spec/jobs/check_consistency_job_spec.rb

```diff
@@ -0,0 +1,47 @@
+# frozen_string_literal: true
+
+require "rails_helper"
+
+module Dttp
+  describe CheckConsistencyJob do
+    let(:trainee) { create(:trainee, id: 1) }
+    let(:contact) { double({ updated_at: Faker::Date.backward(days: 2) }) }
+    let(:placement_assignment) { double({ updated_at: Faker::Date.backward(days: 2) }) }
+    subject { described_class.perform_now(consistency_check.id) }
+
+    before do
+      allow(Dttp::Contacts::Fetch).to receive(:call).with({ trainee: trainee }) { contact }
+      allow(Dttp::PlacementAssignments::Fetch).to receive(:call).with({ placement_assignment_dttp_id: trainee.placement_assignment_dttp_id }) { placement_assignment }
+      allow(SlackNotifierService).to receive(:call) { "foobar" }
+      allow(Trainee).to receive(:find).with(trainee.id) { trainee }
+    end
+
+    context "in sync" do
+      let(:consistency_check) do
+        create(:consistency_check,
+               trainee_id: trainee.id,
+               contact_last_updated_at: contact.updated_at,
+               placement_assignment_last_updated_at: placement_assignment.updated_at)
+      end
+
+      it "it will not do anything" do
```

Comment:
Can we reword these? We mostly follow https://www.betterspecs.org in this regard... So... `it`is written in third person present tense e.g. here `it "does nothing"`, or below - `it "notifies slack"`. Context has where or when e.g. `context "when in sync"`, `context "when out of sync"`.

Reply:
All done 👍